uxkit-youtube/src/main/java/com/voxeet/uxkit/youtube/YoutubeMediaPresentationProvider.java

```diff
@@ -5,42 +5,33 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
+import com.voxeet.uxkit.common.logging.ShortLogger;
+import com.voxeet.uxkit.common.UXKitLogger;
 import com.voxeet.uxkit.presentation.provider.AbstractMediaPlayerProvider;
 
 /**
  * Manage and create youtube videos
  */
 public class YoutubeMediaPresentationProvider extends AbstractMediaPlayerProvider<YoutubeMediaPresentationView> {
 
-    @Nullable
-    private static String youtubeKey;
+    private final static ShortLogger Log = UXKitLogger.createLogger(YoutubeMediaPresentationProvider.class);
```

Comment:
Since it's a final static, it should be LOG but due to its purpose, mimicking the default android.util.Log singleton, Log is more meaningful

Reply:
But we should not mimic a singleton if it's local variable and is seen as one in IDE. Upper case is commonly used in defining constants like `SOMETHING = 10` and enums, not for class instances like our `log`.

Reply:
editing my above comment, Log is not a singleton but a static provider. Anyway uppercase is a standard for any kind of static final. Since you mentioned translating to Kotlin, even the Kotlin translation layer is following this for Java callers when Singleton are used from object Foo {}

Since here LOG or log provide way less value than Log, this is why it's being used. As you mentioned as well indirectly, there are times things we provide are prevalent onto what can be written

Reply:
It is not standard for any kind of static final but whatever.